### ISSUE TITLE
Can this package convert ICNS to ICO and vice versa? Additionally, can it convert ICNS to an array of PNG files?

### ISSUE THREAD

**Issue by futuremotiondev** [idesis-gmbh/png2icons#8]
I'm trying to understand the flexibility of this module. I need a solution that will convert an `ICNS` file either directly to an `ICO`, or to a set of PNG files at the embedded resolutions from the `ICNS` file.

Is this possible? Thanks much.

**Comment by mn4367** [comment]
No, unfortunately that's not possible. Maybe I'll add this feature in the future so I'll leave this issue open.

**Comment by rollingmoai** [comment]
> or to a set of PNG files at the embedded resolutions from the `ICNS` file.

Try out https://github.com/JackMordaunt/icns. Additionally, the latest development version of GIMP (`2.99.14`) has added support for the ICNS format.
